codelists/open/impact_metric.csv

```diff
@@ -1,11 +1,19 @@
-Code,Title
-damage ratio,Damage ratio
-mean damage ratio,Mean damage ratio
-probability of exceeding damage limit state,Probability of exceeding damage limit state
-loss ratio,Loss ratio
-total loss,total loss
-number of casualties(fatalities),Number of casualties (fatalities)
-casualty(fatality) ratio,Casulalty (fatality) ratio
-annual average loss,Annual average loss
-downtime,Downtime
-damage index,Damage index
+Code,Title,Definition,Category
+damage_ratio,Damage ratio,The estimated repair cost of an asset divided by the total replacement cost of the asset [0-n] used in vulnerability functions.,Vulnerability
+mean_damage_ratio,Mean damage ratio,The mean value in a distribution of estimated repair costs divided by the total replacement cost of an asset [0-n] used in vulnerability functions.,Vulnerability
+probability,Probability,The probability of reaching or exceeding a damage limit state specified in a fragility function [%].,Vulnerability
+damage_index,Damage index,"A specified index or classification of damage levels, which may be numeric or non-numeric.",Vulnerability
+loss_ratio,Loss ratio,"The loss to an asset divided by the total replacement cost of the asset [0-n] as a function of damage state, used in damage-to-loss-models.",Vulnerability
+mean_loss_ratio,Mean Loss Ratio,"The mean loss ratio in a distribution of losses divided by the total replacement cost of the asset [0-n] as a function of damage state, used in damage-to-loss-models.",Vulnerability
+downtime,Downtime,"Duration of downtime, used in vulnerability modelling of business or service interruption.",Vulnerability
+casualty_ratio,Casualty ratio,The number of expected casualties or fatalities as a proportion of total inhabitants of and asset or population [0-1] used in casualty or mortality functions.,Vulnerability
+economic_loss,Total Loss,"The monetary value of estimated economic impact or total loss (ground up loss in insurance terminology). Impact_type denotes whether this refers to total, direct, or indirect loss.",Loss
+insured_loss,Total Loss,The monetary value of estimated insured losses.,Loss
+loss_ratio,Loss Ratio,"Estimated loss as a proportion of total replacement cost or insured value. Impact_type denotes whether this refers to total, direct, or indirect loss.",Loss
+number_casualties,Number of casualties (fatalities),The estimated number of casualties or fatalities.,Loss
+casualty_ratio,Casualty ratio,Proportion of estimated casualties or fatalities in the total population.,Loss
+annual_average_loss,Annual average loss,"Expected loss over a one-year time period according to long-term averages. Impact_type denotes whether this refers to total, direct, or indirect loss.",Loss
+probable_maximum_loss,Probable Maximum Loss,Estimated maximum loss occurring at a given exceedance probability or return period.,Loss
```

Comment:
@stufraser1 are these expected to be monetary values? If so we should suffix the codes with `_value`. Also suggest reorder the components of the code to match the structure of other codes:

```suggestion
loss_annual_average_value,Annual average loss,"Expected loss over a one-year time period according to long-term averages. Impact_type denotes whether this refers to total, direct, or indirect loss.",Loss
loss_probable_maximum_value,Probable Maximum Loss,Estimated maximum loss occurring at a given exceedance probability or return period.,Loss
```